### ISSUE TITLE
404 Not found

### ISSUE THREAD

**Issue by macalinao** [frugalmechanic/fm-sbt-s3-resolver#39]
Getting an error like this whenever I run `sbt publish`:

```
[error] (*:publish) com.amazonaws.services.s3.model.AmazonS3Exception: Not Found (Service: Amazon S3; Status Code: 404; Error Code: 404 Not Found; 
Request ID: --redacted--), S3 Extended Request ID: --redacted--
```

Any ideas?

**Comment by tpunder** [comment]
What version of SBT?

**Comment by tpunder** [comment]
This should be fixed in version `0.13.0` via https://github.com/frugalmechanic/fm-sbt-s3-resolver/commit/b41cda379c9602a5801adcb2a9de8585709b154d

**Comment by macalinao** [comment]
Sorry for not responding. Thanks for looking into this!
